recipe/meta.yaml

```
@@ -21,11 +19,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/openvinotoolkit/openvino_tokenizers/archive/refs/tags/{{ version }}.0.tar.gz
-  sha256: d2fd7f5023f7d26072c2248a98520e2b8cb45f6d202a69392a7a922bf5a27951
+  url: https://github.com/ilya-lavrenov/openvino_tokenizers/archive/72f87084983fc9fcf84ed5d131dd2ef15ff2af5d.tar.gz
```

lets build it from the release branch
```suggestion
  url: https://github.com/ilya-lavrenov/openvino_tokenizers/archive/72f87084983fc9fcf84ed5d131dd2ef15ff2af5d.tar.gz
```